code/modules/mob/living/simple_animal/distortion/legend/papa_bongy.dm

```diff
@@ -0,0 +1,332 @@
+					say("BUGAAAAAAAAAAAAAAAAAAAAAAAAWK!")
+					to_chat(user, "[src] grimaces, sputters and coughs! It begins thrashing about, and you feel the temperature around you rise...")
+					new /mob/living/simple_animal/hostile/distortion/papa_bongy/spicy(get_turf(src))
+					qdel(src)
+					break
+			if(R.type in yucky_list)
+				say("BAWK BAWK!!!")
+				to_chat(user, "[src] didn't like that at all.")
+				adjustBruteLoss(150)
+				break
+			happy = TRUE
+	if(istype(I, /obj/item/food/pizzaslice))
+		if(I.type in bad_pizza_list)
+			say("BAWK BAWK!!!")
+			to_chat(user, "[src] didn't like that at all.")
+			adjustBruteLoss(150)
+			qdel(I)
+			return
+		happy = TRUE
+	if(happy == FALSE)
+		qdel(I)
+		return
+	else
+		qdel(I)
+		can_act = FALSE
+		addtimer(CALLBACK(src, PROC_REF(Unmanifest)),3 SECONDS)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/OpenFire(atom/A)
+	if(!can_act)
+		return FALSE
+	if(get_dist(src, A) <= 2) //no shooty in mele
+		return FALSE
+	if(LAZYLEN(unit_list) >= units_max)
+		return FALSE
+	ThrowBongy(A)
+	return ..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/Move()
+	if(!can_act)
+		return FALSE
+	..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/CanAttack(atom/the_target)
+	if(ishuman(the_target))
+		var/mob/living/carbon/human/L = the_target
+		if(L.sanity_lost && L.stat != DEAD)
+			if(HAS_AI_CONTROLLER_TYPE(L, /datum/ai_controller/insane/murder/bongy))
+				return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/AttackingTarget()
+	if(!can_act || !target)
+		return FALSE
+	if(!isliving(target))
+		return ..()
+	var/mob/living/carbon/human/H = target
+	if(ishuman(H))
+		if(istype(target.ai_controller, /datum/ai_controller/insane/murder/bongy))
+			LoseTarget()
+			return//need to test whether this is still needed
+		H.add_movespeed_modifier(/datum/movespeed_modifier/clowned)
+		addtimer(CALLBACK(H, TYPE_PROC_REF(/mob, remove_movespeed_modifier), /datum/movespeed_modifier/clowned), 2 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
+		if(H.sanity_lost)
+			BongyPanic(H)
+	..()
+	can_act = FALSE
+	return DingAttack(target)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/proc/DingAttack(target)
+	var/turf/target_turf = get_turf(target)
+	var/list/been_hit = list()
+	var/picked_sound = 'sound/distortions/papa_bongy/PapaBongy_Coin2.ogg'
+	for (var/i = 1, i <= 3, i++)
+		if(stat == DEAD)
+			return
+		sleep(ding_delay)
+		switch(i)
+			if(1)
+				picked_sound = 'sound/distortions/papa_bongy/PapaBongy_Coin2.ogg'
+			if(2)
+				picked_sound = 'sound/distortions/papa_bongy/PapaBongy_Coin3.ogg'
+			if(3)
+				picked_sound = 'sound/distortions/papa_bongy/PapaBongy_Coin4.ogg'
+		playsound(src, picked_sound, 70, FALSE)
+		do_attack_animation(target_turf)
+		for(var/turf/T in range(i, target_turf))
+			new /obj/effect/temp_visual/smash_effect(T)
+			been_hit = HurtInTurf(T, been_hit, 15, WHITE_DAMAGE, check_faction = TRUE, hurt_mechs = TRUE, mech_damage = 15)
```

Comment:
Given that there is a spicy version of this boss, why not make the damage dealt a variable?